back/spec/acceptance/events_spec.rb

```diff
@@ -21,6 +21,13 @@
     parameter :start_at_lt, 'Filter by maximum start at', type: :string
     parameter :start_at_gteq, 'Filter by minimum start at', type: :string
     parameter :project_publication_statuses, 'The publication statuses of the project to filter events by', type: :array
+    parameter :ongoing_during, <<~DESC, required: false, type: :string
+      Filter events by date range. Only returns events that are ongoing during the 
+      specified date range, meaning events that overlap with the given range. The date 
+      range should be provided as a string in the format "[<start_date>,<end_date>]", 
+      where <start_date> and <end_date> are ISO 8601 dates. 'null' can be used for 
+      open-ended ranges.
+    DESC
```

Comment:
I will propose to standardise date filter parameters (with this format or another) during the next CDM.